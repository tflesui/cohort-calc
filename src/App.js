// import logo from './logo.svg';
import './App.css';

import { Button, Container } from 'reactstrap';

function App() {
  return (
    <div className="App">
      <Container style={{ marginTop: 20 }}>
        <Button color="success">Lessgo!</Button>
      </Container>
    </div>
  );
}

export default App;
