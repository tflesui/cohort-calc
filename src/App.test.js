import { cleanup, render } from '@testing-library/react';
import App from './App';

afterEach(cleanup)

test('should take a snapshot', () => {
  const { asFragment } = render(<App />)
  
  expect(asFragment(<App />)).toMatchSnapshot()
});
