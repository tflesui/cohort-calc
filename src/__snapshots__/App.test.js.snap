// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="text-center container"
    >
      <h1>
        Cohort Calculator
      </h1>
      <div
        class="jumbotron jumbotron-fluid"
      >
        <div
          class="container"
          style="margin-top: 20px;"
        >
          <p
            class="text-primary"
            data-testid="hrsCounter"
          >
            Max Instructor Hours: 0 
          </p>
          <button
            class="btn btn-success"
            data-testid="increase-button"
            style="margin-right: 5px;"
            type="button"
          >
            Increase
          </button>
        </div>
        <form
          class=""
          style="margin: 10px;"
        >
          <div
            class="row form-group"
          >
            <label
              class="col-sm-2 col-form-label"
              for="studentName"
            >
              Student Name
            </label>
            <div
              class="col-sm-4"
            >
              <input
                class="form-control"
                id="studentName"
                name="name"
                placeholder="Student Name"
                required=""
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            class="row form-group"
          >
            <label
              class="col-sm-2 col-form-label"
              for="earnings"
            >
              Earnings Potential
            </label>
            <div
              class="col-sm-4"
            >
              <input
                class="form-control"
                id="earnings"
                name="earnings"
                placeholder="0"
                required=""
                type="number"
                value="0"
              />
            </div>
          </div>
          <div
            class="row form-group"
          >
            <label
              class="col-sm-2 col-form-label"
              for="hours"
            >
              Hours Needed
            </label>
            <div
              class="col-sm-4"
            >
              <input
                class="form-control"
                id="hoursNeeded"
                name="hoursNeeded"
                placeholder="0"
                required=""
                type="number"
                value="0"
              />
            </div>
          </div>
          <div
            class="form-group"
          >
            <button
              class="btn btn-primary"
              style="margin-right: 5px;"
              type="submit"
            >
              Add Student
            </button>
            <button
              class="btn btn-secondary"
              style="margin-left: 5px;"
              type="button"
            >
              Clear Form
            </button>
          </div>
        </form>
        <table
          class="table table-striped"
        >
          <thead>
            <tr>
              <th>
                Name
              </th>
              <th>
                Earnings Potential
              </th>
              <th>
                Instruction Hours Needed
              </th>
              <th>
                Action
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>
                Jane
              </td>
              <td>
                $1,000
              </td>
              <td>
                3
              </td>
              <td>
                <button
                  class="btn btn-danger"
                  type="button"
                >
                  Remove
                </button>
              </td>
            </tr>
            <tr>
              <td>
                Bob
              </td>
              <td>
                $3,000
              </td>
              <td>
                5
              </td>
              <td>
                <button
                  class="btn btn-danger"
                  type="button"
                >
                  Remove
                </button>
              </td>
            </tr>
          </tbody>
        </table>
        <div />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
